src/core/linter-rules/check-punctuation.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Linter rule "check-punctuation". Makes sure the there are no punctuations missing at the end of a <p>
+ *   in the ReSpec config.
+ */
+import { lang as defaultLang } from "core/l10n";
+import LinterRule from "core/LinterRule";
+
+const name = "check-punctuation";
+const punctuationMarks = [".", ":", "!", "?"];
+
+const meta = {
+  en: {
+    description: "`<p>` tags should end with either of ., :, !, ? ",
```

Comment:
See my previous comment about using the `punctuationMark` here via a template string. Otherwise, we would need to update this in multiple places.